lib/ecl/ecl_file_view.cpp

```diff
@@ -200,11 +202,12 @@ int ecl_file_view_find_kw_value( const ecl_file_view_type * ecl_file_view , cons
 }
 
 const char * ecl_file_view_iget_distinct_kw( const ecl_file_view_type * ecl_file_view , int index) {
-  return stringlist_iget( ecl_file_view->distinct_kw , index);
+  const std::string& string = ecl_file_view->distinct_kw->operator[](index);
```

Comment:
This is perfectly fine, but I think `ecl_file_view->distinct_kw[index]` looks a bit better

Reply:
I fully agree  - it is the same argument as above.